packages/w3up-client/src/client.js

```diff
@@ -124,17 +124,19 @@ export class Client extends Base {
   /**
    * Use a specific space.
    *
-   * @param {import('./types').DID<'key'>} did
```

Comment:
reverting this ts api interface change https://github.com/web3-storage/w3up-client/commit/ee1cf3a30a79a98c21f2e897d6a26e443b41390f#r106486919

(let's introduce the `DID` -> `DID<'key'>` again in one fell swoop that also adjusts some `@web3-storage/access/agent-data` types that still need to be done)

Reply:
oops this was not 'releasable' due to using `test:` conventional commit

Reply:
This change went out in w3up-client@5.5.0